### ISSUE TITLE
[intro-to-python] Part 7 - Loading Image Data -  No such file or directory: 'Cat_Dog_data/train'

### ISSUE THREAD

**Issue by tianyu0523** [udacity/deep-learning-v2-pytorch#139]
![image](https://user-images.githubusercontent.com/25603473/52584003-0948dc80-2e42-11e9-8ba3-94a54064719a.png)
The message shows that 'No such file or directory: 'Cat_Dog_data/train''. Did anyone meet or solve this problem?

**Comment by jchernus** [comment]
I can't replicate this on my end, my notebook is able to see the file. Anyone else? Would a refresh/workspace reset fix it?

**Comment by ruqianchen** [comment]
The notebook says "You can download the dataset already structured like this from here. I've also split it into a training set and test set." You need to follow the following link to download and unzip the files.

The link provided is `https://s3.amazonaws.com/content.udacity-data.com/nd089/Cat_Dog_data.zip`

**Comment by fabrizioschiano** [comment]
I had the same problem and downloading the dataset from the link given by @ruqianchen solved my problem.
